src/routes/index.js

```diff
@@ -1,26 +1,14 @@
+/* eslint-disable linebreak-style */
+import dotenv from 'dotenv';
 import { Router } from 'express';
-// import welcomeRoute from './welcome';
-import Badroute from './badroute';
```

Comment:
I think we still need this Badroute in our project.

Reply:
Oh I wasn't aware of that